corehq/apps/analytics/static/analytix/js/appcues.js

```diff
@@ -0,0 +1,60 @@
+/* global Appcues, Array, window */
+
+/**
+ * Instantiates the AppCues analytics and customer support messaging platform.
+ */
+hqDefine('analytix/js/appcues', [
+    'underscore',
+    'analytix/js/initial',
+    'analytix/js/logging',
+    'analytix/js/utils',
+], function (
+    _,
+    initialAnalytics,
+    logging,
+    utils
+) {
+    'use strict';
+    var _get = initialAnalytics.getFn('appcues'),
+        _logger = logging.getLoggerForApi('Appcues'),
+        _ready = $.Deferred(),
+        EVENT_TYPES = {
```

Comment:
Minor, but maybe consider making the labels more human-friendly, like "Added a question", since growth team will be reading them.